File: kmsConvert.js
const converter = () => {
  kms = prompt('Enter the number of kilometers you would like to convert to miles');
  const divisor = 1.6094;
  miles = kms / divisor;
  miles = (miles.toString());
  console.log('You want to convert ' + kms + ' kms .');
  console.log('If you convert ' + kms + ' kms to miles you will get ' + miles + ' miles.');
};

converter();

File: kmsConvert.rb
def converter
puts('Enter the nuber of kilometer you would like to convert to miles ')
kms = gets().to_f
divisor= 1.60934
convert=kms/divisor
miles = convert.to_s
puts("ok you said #{kms} kms.")
puts("If you convert #{kms} kms to miles you will get #{miles} miles")
end

converter()
